src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Network/NewAzureReservedIP.cs

```diff
@@ -93,6 +94,14 @@ public string Slot
             set;
         }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, ParameterSetName = ReserveNewIPParamSet, HelpMessage = "List of IPTags.")]
+        [ValidateNotNullOrEmpty]
+        public List<IPTag> IPTags
```

Comment:
@skomarag please rename this parameter to `IPTag` - having a singular noun for the parameter name follows the [PowerShell strongly encouraged developer guidelines](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#singular-noun-1)